src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarBidAction.tsx

```diff
@@ -103,20 +103,16 @@ export class ArtworkSidebarBidAction extends React.Component<
           {({ user }) => {
             return (
               <Box>
-                {artwork.sale.is_registration_closed && !registeredToBid ? (
-                  <Button width="100%" size="large" mt={1} disabled>
-                    Registration closed
-                  </Button>
-                ) : (
-                  <Button
-                    width="100%"
-                    size="large"
-                    mt={1}
-                    onClick={() => this.redirectToLiveBidding(user)}
-                  >
-                    Enter live bidding
-                  </Button>
-                )}
+                <Button
+                  width="100%"
+                  size="large"
+                  mt={1}
+                  onClick={() => this.redirectToLiveBidding(user)}
+                >
+                  {artwork.sale.is_registration_closed && !registeredToBid
+                    ? "Watch live bidding"
+                    : "Enter live bidding"}
```

Comment:
@oxaudo - now that we're near completion we should make sure there are tests for these little bits. Can merge as is but lets follow up in a cleanup pass.